src/StudioComponents/Util/themeProvider.lua

```diff
@@ -84,15 +85,19 @@ local function updateTheme()
 		end
 	end
 	themeProvider.Theme:set(Studio.Theme.Name)
+
+	local _,_,v = Studio.Theme:GetColor(Enum.StudioStyleGuideColor.MainBackground):ToHSV()
+	themeProvider.IsDark:set(v<=0.6)
 end
+updateTheme()
```

Comment:
Could probably be inside the do statement at the bottom so that way it's easier to notice.